tests/yast2_gui/yast2_hostnames.pm

```diff
@@ -21,10 +21,18 @@ use testapi;
 use utils 'clear_console';
 use x11utils qw(turn_off_kde_screensaver turn_off_gnome_screensaver);
 
+
+# Enable additional arguments for nested calls of wait_still_screen
+# SLOW_TYPING_SPEED == max_interval => 13
+sub type_string_extended {
```

Comment:
What's about moving it to some lib, for example to `utils` where `type_string_slow` is defined. I've suggested extracting this method, so we can reuse it not only in this test module. For single module we could have put parameters to the hash and simply reuse it all 3 calls.

Reply:
Well, I thought that we have agreed on leaving it for now, since the parameters might not be suitable for osd or o3. Nevertheless I will extract it

Reply:
Yeah, that's why I've merged PR with fix to yast2_hostnames. But you've created this PR, so I thought we address it now then =).